### ISSUE TITLE
SIGABRT after loading savegame

### ISSUE THREAD

**Issue by TheCrazyT** [colobot/colobot#1448]
This happens directly after loading a savegame.
(Zipped savegame-folder https://www.dropbox.com/s/a9jszteie463dna/save2107210917Vulkania2.zip?dl=1 )

```
Program: D:\Program Files\colobot\colobot.exe
File: C:/Users/Simbax/Documents/colobot/src/object/implementation/programmable_impl.cpp, Line 158

Expression: op < static_cast<int>(dynamic_cast<CProgramStorageObject*>(m_object)->GetProgramCount())

Unhandled exception occurred!
==============================
SIGABRT, abort
==============================

This is usually caused by a bug. Please report this on http://github.com/colobot/colobot/issues
including information on what you were doing before this happened and all the information below.
==============================
You are running official 0.1.12-alpha build.

The game was in phase PHASE_SIMUL (ID=7)
Last started level was: category=missions chap=6 rank=2
==============================
```

**Comment by TheCrazyT** [comment]
https://github.com/colobot/colobot/blob/master/src/object/implementation/programmable_impl.cpp#L158
There is something wrong with the "ProgramStorage".


Additional info:
I didn't have problems loading on earlier levels.
I was also able to load an earlier savegame on the same level.
Not shure whats suddenly different, because I didn't write or add any robot programm between the 2 savegames.
Also have an auto-savegame about 2 minutes early, but it crashes there as well.
Actions that I did before the save was repair and reload batteries (but that probably has nothing to do with the crash).
Programs on other robots that currently run are:
- collect and convert titanium (non-modified version of that level)
- refresh batteries on towers (non-modified version of that level)

**Comment by TheCrazyT** [comment]
Hm now this is strange ... I tried to leard earlier savegames on same level but I'm now getting the same error.
Although I'm quite shure that there was one working savegame.
Maybe it doesn't necessarily have something to do with the savegame itself. (but I have no idea what else could be the problem)
Strangely ... loading savegames of different levels work.

**Comment by melex750** [comment]
Looking at the ```data.sav``` file, this is probably a duplicate of #1370. And won't crash the game on ```dev``` branch because
#1253 was merged.  Probably fixed in #1440.

**Comment by TheCrazyT** [comment]
@melex750 

you are correct, it works with the version under:
https://github.com/colobot/colobot/actions/runs/972949061
